inductiva/resources/machine_groups.py

```diff
@@ -69,13 +69,25 @@ def _machine_group_list_to_str(machine_group_list) -> str:
             spot, machine_group.create_time
         ])
 
-    formatters = {"Started at (UTC)": format_utils.datetime_formatter}
-
-    return format_utils.get_tabular_str(
-        rows,
-        columns,
-        formatters=formatters,
-    )
+    formatters = {
+        "Started at (UTC)": [format_utils.datetime_formatter],
```

Comment:
Could the spacing formatter be added as a global formatter to the table? It starts to be painful adding in each place. Wdyt?

Reply:
We found a way around this. Removed in the next commit